api/reservation_units_api.py

```diff
@@ -25,10 +26,17 @@ class ReservationUnitFilter(filters.FilterSet):
     )
 
 
+class ReservationUnitSerializer(serializers.ModelSerializer):
```

Comment:
Duplicate class name, might get a bit confusing, should probably be (reservationunit)imageserializer

Reply:
Yes, accident. Will fix.